lib/util/package-rules/index.spec.ts

```diff
@@ -43,6 +43,7 @@ describe('util/package-rules/index', () => {
       depName: 'a',
       isBump: true,
       currentValue: '1.0.0',
+      labels: ['bump'],
```

Comment:
```suggestion
```

Change https://github.com/renovatebot/renovate/pull/26880/files#diff-b9fa5327dbf0d50ad7eb9d30c074f7cae334fc05bae58698e8b70206e9d29d14R60 instead to include the labels, this way you are validating that changes were applied